psydac/feec/tests/test_global_projectors.py

```diff
@@ -237,18 +250,19 @@ def test_derham_projector_2d_hcurl(ncells, degree, periodic):
 @pytest.mark.parametrize('ncells', [[30,30,30]])
 @pytest.mark.parametrize('degree', [[2,2,2], [2,3,2], [3,3,3]])
 @pytest.mark.parametrize('periodic', [[False, False, False], [True, True, True]])
+@pytest.mark.parametrize('multiplicity', [[1,1,1],[1,2,2],[2,2,2]])
 
-def test_derham_projector_3d(ncells, degree, periodic):
+def test_derham_projector_3d(ncells, degree, periodic, multiplicity):
 
     domain = Cube('Omega', bounds1 = (0,2*np.pi), bounds2 = (0,2*np.pi), bounds3 = (0,2*np.pi))
     domain_h = discretize(domain, ncells=ncells, periodic=periodic)
     
     derham   = Derham(domain)
-    derham_h   = discretize(derham, domain_h, degree=degree, get_H1vec_space = True)
+    #change mulitplicity if higher than degree to avoid problems (case p<m doesn't work)
+    multiplicity = [min(m,p) for p, m in zip (degree, multiplicity)]
```

Comment:
```suggestion
    #change multiplicity if higher than degree to avoid problems (case p<m doesn't work)
    multiplicity = [min(m, p) for p, m in zip(degree, multiplicity)]

```